server/settings.py

```diff
@@ -124,6 +124,10 @@ def env(variable, fallback_value=None):
         'task': 'apps.publish.content_purge',
         'schedule': crontab(minute=30)
     },
+    'messaging:remove_expired': {
+        'task': 'apps.messaging.purge_chat_sessions',
+        'schedule': crontab(hour=0, minute=0)
```

Comment:
In Cron Expression all intervals start from 0. In this case, the job runs every day midnight 1200 AM.